src/ui/SmartSnippet/UserFeedbackBanner.ts

```diff
@@ -148,7 +163,9 @@ export class UserFeedbackBanner {
     }
     this.isUseful = isUseful ? UsefulState.Yes : UsefulState.No;
     this.yesButton.classList.toggle(BUTTON_ACTIVE_CLASSNAME, isUseful);
+    this.yesButton.setAttribute('aria-pressed', `${isUseful}`);
```

Comment:
What does the `aria-pressed` do exactly ? 

Does it signify to screen reader "here's what going to happen when you click this button" ?

Reply:
When the `aria-pressed` attribute exists, whether `true` or `false`, it signifies the element is a toggleable button.